code/game/machinery/vending_types.dm

```diff
@@ -506,11 +506,13 @@
 		/obj/item/weapon/reagent_containers/food/drinks/drinkingglass = 8,
 		/obj/item/clothing/suit/chef/classic = 2,
 		/obj/item/weapon/reagent_containers/cooking_container/oven = 5,
-		/obj/item/weapon/reagent_containers/cooking_container/fryer = 4
+		/obj/item/weapon/reagent_containers/cooking_container/fryer = 4,
+		/obj/item/weapon/storage/toolbox/lunchbox/nt = 6
 	)
 	contraband = list(
 		/obj/item/weapon/material/kitchen/rollingpin = 2,
```

Comment:
why is the rolling pin in the contraband list in the first place?

Reply:
Not changed in this PR, maybe ask in the code dungeon?